Python/samples/house-energy/sim/house_simulator.py

```diff
@@ -2,7 +2,21 @@
 import matplotlib.pyplot as plt
 
 class House():
-    def __init__(self, K: float=0.5, C: float=0.3, Qhvac: float=9, hvacON: float=0, occupancy: float=1, Tin_initial: float=30):
+    def __init__(self, 
+                K: float=0.5, 
+                C: float=0.3, 
+                Qhvac: float=9, 
+                hvacON: float=0, 
+                occupancy: float=1, 
+                Tin_initial: float=30, 
+                Tout_initial: float= 20,
```

Comment:
How are you honoring `Tout_initial` with the sinewave?

Reply:
oh Tout_initial is the median name i chose. We can alter this to initial if we want in the Signal builder. But i ask that we be strict about `key_param` for now.